src/encord_active/lib/encord/actions.py

```diff
@@ -333,15 +336,26 @@ def create_project(
         self.project_meta["project_hash"] = new_project.project_hash
         update_project_meta(self.project_file_structure.project_dir, self.project_meta)
 
+        # Reverse ordering : correct data unit hash mapping data
+        data_hash_mapping = DataHashMapping()
+        for new_du, old_du in dataset_creation_result.new_du_hash_to_original_mapping.items():
+            data_hash_mapping.set(old_du, new_du)
+
         replace_uids(
             self.project_file_structure,
-            dataset_creation_result.lr_du_mapping,
+            dataset_creation_result.lr_du_mapping,  # FIXME: needs to be updated to work correctly for non-images
```

Comment:
i don't think we support image group creation for non-encord projects so i think this is fine.